interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -212,7 +209,8 @@ object SpelExpressionParser extends LazyLogging {
               typeDefinitionSet: TypeDefinitionSet,
               methodExecutionForUnknownAllowed: Boolean,
               dynamicPropertyAccessAllowed: Boolean,
-              spelExpressionExcludeList: SpelExpressionExcludeList)
+              spelExpressionExcludeList: SpelExpressionExcludeList,
+              conversionService: ConversionService = DefaultSpelConversionsProvider.getConversionService)
```

Comment:
I wonder if we should provide default value here? It may happen that we're using this method in > 1 place?